controllers/v1/reports.js

```diff
@@ -58,47 +57,43 @@ module.exports = class Reports {
     * @apiUse errorBody
     */
 
-    /**
-      * Entity Report
-      * @method
-      * @name entity
-      * @param {Object} req - request data.
-      * @param {String} req.params._id - Entity id.
-      * @returns {JSON} enity report details.
-     */
-    async entity(req) {
-        return new Promise(async (resolve, reject) => {
-            try {
-
-                const entityReports = await reportsHelper.entity(
-                    req.params._id,
-                    req.userDetails.userInformation.userId,
-                    req.userDetails.userToken,
-                    req.userDetails.userInformation.userName,
-                    req.query.reportType,
-                    req.query.programId ? req.query.programId : "",
-                    req.query.requestPdf ? (req.query.requestPdf).toLowerCase() =="true" ? true :false : false,
-                    req.headers['x-app-ver']
-                );
-                
-                return resolve({
-                    message: entityReports.message,
-                    result: entityReports.data
-                });
-
-            } catch (error) {
-                return reject({
-                    status: error.status || HTTP_STATUS_CODE.internal_server_error.status,
-                    message: error.message || HTTP_STATUS_CODE.internal_server_error.message,
-                    errorObject: error
-                });
-            }
-        })
-    }
-
-    
-
-    /**
+	/**
+	 * Entity Report
+	 * @method
+	 * @name entity
+	 * @param {Object} req - request data.
+	 * @param {String} req.params._id - Entity id.
+	 * @returns {JSON} enity report details.
+	 */
+	async entity(req) {
+		return new Promise(async (resolve, reject) => {
+			try {
+				const entityReports = await reportsHelper.entity(
+					req.params._id,
+					req.userDetails.userInformation.userId,
+					req.userDetails.userToken,
+					req.userDetails.userInformation.userName,
+					req.query.reportType,
+					req.query.programId ? req.query.programId : '',
+					req.query.requestPdf ? (req.query.requestPdf.toLowerCase() == 'true' ? true : false) : false,
+					req.headers['x-app-ver']
+				)
+
+				return resolve({
+					message: entityReports.message,
+					result: entityReports.data,
+				})
+			} catch (error) {
+				return reject({
+					status: error.status || HTTP_STATUS_CODE.internal_server_error.status,
+					message: error.message || HTTP_STATUS_CODE.internal_server_error.message,
+					errorObject: error,
+				})
+			}
+		})
+	}
+
+	/**
     * @api {post} /improvement-project/api/v1/reports/getProgramsByEntity/:_id
```

Comment:
@MallanagoudaB when you make change, look for similar changes please. correct the url